README.md

```diff
@@ -19,21 +33,22 @@ AWS credentials can be exported as environment variables or permissions can be g
 |backup_dir| Path to directory which will hold the final backup|
 |s3_bucket| S3 bucket to store/retrieve buckets to/from|
 |s3_path| S3 Path prefix, can be used for backing up multiple core nodes to the same bucket|
+|verifying_gpg_key| If you wish to verify the Stellar provided backups, leave this as backtups@stellar.org|
```

Comment:
I don't think we need to have 2 variables to contain the same key_id, I do not foresee users creating/signing their backups and then verifying with a different key_id.